File: 프로그래머스/level1_음양 더하기.py
def solution(absolutes, signs):
    answer = 0
    for i in range(len(absolutes)):
        if signs[i]:
            answer += absolutes[i]
        else:
            answer -= absolutes[i]

    return answer

absolutes = [1, 2, 3]
signs = [False, False, True]
print(solution(absolutes, signs))

File: 프로그래머스 복습(java)/src/level1/level1_음양_더하기.java
package level1;

public class level1_음양_더하기 {
    static class Solution {
        public int solution(int[] absolutes, boolean[] signs) {
            int answer = 0;
            for (int i = 0; i < absolutes.length; i++) {
                if (signs[i]) {
                    answer += absolutes[i];
                } else {
                    answer -= absolutes[i];
                }
            }
            System.out.println("answer = " + answer);
            return answer;
        }
    }

    public static void main(String[] args) {
        Solution solution = new Solution();
        int[] absolutes1 = {4, 7, 12}, absolutes2 = {1, 2, 3};
        boolean[] sign1 = {true, false, true}, sign2 = {false, false, true};
        solution.solution(absolutes1, sign1);
        solution.solution(absolutes2, sign2);
    }
}
